azure-iptables-monitor/iptables_monitor.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	goiptables "github.com/coreos/go-iptables/iptables"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/component-base/logs"
+	"k8s.io/component-base/version/verflag"
+	"k8s.io/klog/v2"
+)
+
+// Version is populated by make during build.
+var version string
+
+var (
+	configPath    = flag.String("input", "/etc/config/", "Name of the directory with the allowed regex files")
```

Comment:
along with the monitor binary will we have to add these allowed regex files? how are the allowed rules determined?

Reply:
yes-- the allowed rules are ones that we have "accounted for"/expect. Once we have mimicked the rule's functionality post-upgrade, we will add it to the allowed rules.